pkg/cover/backend/elf.go

```diff
@@ -148,6 +148,7 @@ func elfReadModuleCoverPoints(module *Module, info *symbolInfo) ([2][]uint64, er
 			// Note: this assumes that call instruction is 1 byte.
 			pc := module.Addr + rel.Off - 1
 			index := int(elf.R_SYM64(rel.Info)) - 1
+			pc -= uint64(arches[target.Arch].opcodeOffset)
```

Comment:
This will invoke map lookup on every iteration (unnecessary performance pessimization).
Please lookup arch before the loop like in the other function.